src/dcd/server/autocomplete/ufcs.d

```diff
@@ -27,6 +26,8 @@ enum string[string] INTEGER_PROMOTIONS = [
         "dchar": "uint",
     ];
 
+enum MAX_RECURSION_DEPTH = 1;
```

Comment:
```suggestion
enum MAX_RECURSION_DEPTH = 50;
```

multiple alias this are valid, let's pick something reasonably high here, instead of 1

Reply:
Done